packages/uniforms-unstyled/__tests__/_createContext.ts

```diff
@@ -32,7 +33,7 @@ export default function createContext(
         showInlineError: false,
         ...context?.state,
       },
-      formRef: {} as BaseForm<unknown>,
-    },
+      formRef: {} as BaseForm<Model>,
+    } as Context<Model>,
```

Comment:
Not a big fan of `as Type`. It can lead to some errors. Maybe instead, we could use `Context<Model>` as a return type of the `createContext`?